robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -167,9 +187,32 @@ function initRosWeb () {
     messageType: 'geometry_msgs/Point'
   })
   battery_temps_listener.subscribe(function (message) {
-    $('#battery-temp-1').text(parseFloat(message.x).toFixed(2))
-    $('#battery-temp-2').text(parseFloat(message.y).toFixed(2))
-    $('#battery-temp-3').text(parseFloat(message.z).toFixed(2))
+    let t1 = parseFloat(message.x).toFixed(2)
+    let t2 = parseFloat(message.y).toFixed(2)
+    let t3 = parseFloat(message.z).toFixed(2)
+
+    $('#battery-temp-1').text(t1)
+    $('#battery-temp-2').text(t2)
+    $('#battery-temp-3').text(t3)
+
+    let temperatures = [t1, t2, t3]
+
+    for (i = 1; i <= temperatures.length ; i++) {
+      let temp = temperatures[i-1]
+      let color = 'white'
+
+      if (temp > MAX_TEMP || temp < MIN_TEMP) {
+        color = 'red'
+        errorSound()
+        if (temp > MAX_TEMP) {
+          navModalMessage('Warning: Battery temperature too high', 'Decreace temperature')
+        } else if (temp < MIN_TEMP) {
+          navModalMessage('Warning: Battery temperature too low', 'Increace temperature')
+        }
```

Comment:
did u want me to use an image or is it fine with a keep fontawesome? :/

Reply:
@adrientremblay  You should keep the threads to one topic only because once the issue is resolve we can close the thread with GitHub.

But I don't mind if you keep it to fontawesome as long as whatever what was used before with 4.7 isnt affected (since you removed it). But it seems to be fine